LPS-151456 Minimize required FDS configuration

markocikos opened the pull request:
### References

- [LPS-151456 Minimize required FDS configuration](https://issues.liferay.com/browse/LPS-151456)
- [FDS technical improvements checklist](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2002681985/FDS+technical+improvements+checklist)
- This is a followup on #2114
- This is partially a followup on #2103

### What is the goal of this PR?

FDS tags have unnecessarily required attributes, that we'd like to make optional, along with defaults. The only required attributes of `headless-display` tag will now be `apiURL` and `id`. We plan to do the same for the `classic-display`, in a separate task.

In addition, we are:
- adding a tab in the sample module with minimal configuration FDS. Intent here is to add all the small features to `customized` tab. If there will be a need for a showcase of a larger disrupting feature, we can add it in additional tabs. Note that both FDS instances are using the same data.
- deprecating unused `portletURL` attribute.
- seting portlet namespace as default `namespace`. This makes it [consistent with `<clay>` tags](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java#L142-L158).

:warning: I fully expect this PR to break our CI tests on the sample module, as we are changing the markup. @Esther-OC @mateusralv

### What does it look like?

![customized](https://user-images.githubusercontent.com/5435511/163381538-976e834c-9edb-4084-bc37-776884e5d09d.png)

![minimum](https://user-images.githubusercontent.com/5435511/163381516-7b0f40d2-2ab3-46d2-a705-30bcbae3d634.png)

### Steps to reproduce

Place sample FDS module on any page.

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/1664/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1664/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-13.liferay.com/job/test-portal-source-format/5044/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-source-format/builds/5044/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/af1608c17ac4af7f70d07ba8a852ec7e7617ba87">af1608c17ac4af7f70d07ba8a852ec7e7617ba87</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/markocikos/liferay-portal/tree/LPS-151456">LPS-151456</a><br/>Branch GIT ID: <a href="https://github.com/markocikos/liferay-portal/commit/42c9d2eea339fa8bb74aba88cc65b18aa1bf11c8">42c9d2eea339fa8bb74aba88cc65b18aa1bf11c8</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-13.liferay.com/job/test-portal-source-format/5044/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-source-format/builds/5044/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-13.liferay.com/job/test-portal-source-format/5044/">test-portal-source-format#5044</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-13.liferay.com/userContent/jobs/test-portal-source-format/builds/5044/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/2124">liferay-frontend#2124</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1666925739">[master] ci:test:sf - markocikos &gt; liferay-frontend - PR#2124 - 2022-04-14[04:30:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-13.liferay.com/job/publish-testray-report/2743/">publish-testray-report#2743</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:stable - 20 out of 26 jobs passed</h3><h3>:x: ci:test:relevant - 20 out of 27 jobs passed in 6 hours 17 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/af1608c17ac4af7f70d07ba8a852ec7e7617ba87">af1608c17ac4af7f70d07ba8a852ec7e7617ba87</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/af1608c17ac4af7f70d07ba8a852ec7e7617ba87">af1608c17ac4af7f70d07ba8a852ec7e7617ba87</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3141">Acceptance Upstream DXP (master) #3141</a></p><details><summary><strong>ci:test:stable - 20 out of 26 jobs PASSED</strong></summary><div><h4>6 Failed Jobs:</h4><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/514571/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489301/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489302/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31591/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489300/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494525/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31590/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345127/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31587/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489172/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19967/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31593/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345130/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494527/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28620/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44914/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31589/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31592/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31588/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345131/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/405112/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494526/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345129/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345128/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28619/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489171/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 20 out of 27 jobs PASSED</strong></summary><div><h4>7 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/1664/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/514571/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489301/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489302/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</strong></strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31591/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</strong></strike></a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489300/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494525/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31590/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345127/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31587/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489172/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19967/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31593/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345130/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494527/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28620/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/44914/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31589/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31592/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31588/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345131/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/405112/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494526/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345129/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/345128/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28619/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489171/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1664/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/514571/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>0 Tests Failed.</p></div><pre><code>Build was aborted</code></pre></div></li><li><div><h5><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489301/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>0 Tests Failed.</p></div><pre><code>Build was aborted</code></pre></div></li><li><div><h5><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489302/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>0 Tests Failed.</p></div><pre><code>Build was aborted</code></pre></div></li><li><div><h5><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/489300/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>0 Tests Failed.</p></div><pre><code>Build was aborted</code></pre></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at af1608c17ac4af7f70d07ba8a852ec7e7617ba87:</strong></summary><ol><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494525/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0/494525//consoleText">AXIS_VARIABLE=0 #494525</a><div><p>Please fix <strong>semantic versioning</strong> on <strong><a href="https://github.com/liferay-frontend/liferay-portal/pull/2124">markocikos/LPS-151456</a></strong><pre><code>     [exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.dao.orm.hibernate               MINOR      6.0.0      6.0.0      6.1.0      VERSION INCREASE REQUIRED
     [exec]   &lt;   class      com.liferay.portal.dao.orm.hibernate.HSQLDialect
     [exec]     +   method     getLimitHandler()
     [exec]       +   return     org.hibernate.dialect.pagination.LimitHandler
     [exec] [Baseline Warning] Bundle Version Change Recommended: 34.1.0
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-impl/portal-impl.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.impl/34.0.0/ee81e8a40a82b6d34a87fb54356e75afbe5d67b/com.liferay.portal.impl-34.0.0.jar</code></pre></p></div></div></li></ol></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/1664/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/1664/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/1664/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

diegonvs commented:
ci:test:relevant

diegonvs commented on modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/BaseDisplayTag.java:
```diff
@@ -129,6 +154,8 @@ protected Map<String, Object> prepareProps(Map<String, Object> props) {
 			props.put("additionalProps", _additionalProps);
 		}
 
+		props.put("namespace", getNamespace());
```
Hey @markocikos, I don't know what namespace is being used for on FDS but If this should be unique for every FDS displayed on the page. I think we should append a string like UNNAMED_FDS or UNREGISTRED_FDS.

beltranrengifo replied:
A random string was used in the previous version, not sure if it's the way...

Do we need the `_randomNamespace` variable and related methods now?

markocikos replied:
`namespace` does not need to be unique for every FDS. `id` needs to be unique, nothing else. If there is something inside FDS instance that needs to generate a unique value, it should probably be a derivative of `id`.

`namespace` refers to **portlet** namespace. This is consistent over basically all tags in DXP. We can easily have multiple FDS instances in a portlet, as portlets tend to have multiple pages.

Within a DXP tag, namespace is typically required to
1. Build parameters for DXP requests
2. Access DOM elements in a portlets that have been set with `<portlet:namespace />`

Looking at master, `randomNamespace` is not used anywhere :thinking: I think we can safely deprecate it.

markocikos replied:
On a closer inspection, we are using `randomNamespace` [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/BaseDisplayTag.java#L128). Admittedly, I don't see a purpose in setting `id` of this `div`, but let's leave this for a separate task.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/3021/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/jenkins-report.html">here</a>.</b></p>

diegonvs commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/minimum.jsp:
```diff
@@ -0,0 +1,26 @@
+<frontend-data-set:headless-display
+	apiURL="<%= fdsSampleDisplayContext.getAPIURL() %>"
+	id="<%= FDSSampleFDSNames.MINIMUM_FDS_SAMPLE %>"
+/>
```
I liked this idea :)

I think we can emphasize in a text that this page will render a FDS with only required properties

markocikos replied:
I think that tab name is enough. There is only one FDS per tab; if there were several, title per FDS would be useful. A more verbose description in UI might make sense, but that's a slippery slope. If we start with that, we should consistently add description in UI of all tabs. For 'customized' tag, that would have something like 10 lines, cluttering up the page.

If we want a detailed explanation of sample module, what we can do is open a document for this in confluence.

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 26 out of 26 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 27 out of 27 jobs passed in 1 hour 36 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1b11fa609d280aadcab65cc6a98bcb0eb3ca3d77">1b11fa609d280aadcab65cc6a98bcb0eb3ca3d77</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0ffabdfa5aa684b913ffd425b16d2f9b4ac27bd1">0ffabdfa5aa684b913ffd425b16d2f9b4ac27bd1</a><br/>Jenkins Build URL: <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)//3143">Acceptance Upstream DXP (master) #3143</a></p><details><summary><strong>ci:test:stable - 26 out of 26 jobs PASSED</strong></summary><details><summary><strong>26 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45644/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431939/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28085/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494551/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342481/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28083/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20070/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33935/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376080/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20072/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28081/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/244034/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28710/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45645/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31672/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45643/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/49111/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47680/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28084/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28086/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/49114/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47837/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47681/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41888/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20068/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376082/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 27 out of 27 jobs PASSED</strong></summary><details><summary><strong>27 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/3021/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45644/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431939/">test-portal-acceptance-pullrequest-batch(master)/empty-osgi-core-dir-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28085/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/494551/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342481/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28083/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly180-mariadb104-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20070/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/33935/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376080/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20072/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28081/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/244034/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28710/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45645/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31672/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/45643/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/49111/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47680/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28084/">test-portal-acceptance-pullrequest-batch(master)/patching-tool-jdk8/0</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28086/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/49114/">test-portal-acceptance-pullrequest-batch(master)/portal-license-smoke-jdk8/0</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47837/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/47681/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/41888/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20068/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/376082/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable/0</a></li></ul></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/jenkins-report.html">here</a>.</h5></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/3021/">test-portal-acceptance-pullrequest(master)#3021</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3021/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/2124">liferay-frontend#2124</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=1667150507">[master] ci:test:relevant - markocikos &gt; liferay-frontend - PR#2124 - 2022-04-14[11:28:43]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/8556/">publish-testray-report#8556</a><br/></div></div>

beltranrengifo commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/MinimumSampleTableFDSView.java:
```diff
@@ -0,0 +1,64 @@
+		FDSTableSchemaBuilder fdsTableSchemaBuilder =
+			_fdsTableSchemaBuilderFactory.create();
+
+		fdsTableSchemaBuilder.addFDSTableSchemaField("id", "id");
```
Maybe we can also remove some columns in this sample? The `ID` and `Author` for example?

markocikos replied:
I think there is some value in keeping the same fields. For example, you can see at glance the difference between default `Author` and `Author` with the custom renderer.

I tried to do is to remove `status` renderer from the status field, to keep consistent default renderer on all fields, but that breaks the component. This is a can of worms we can open in a separate task.

beltranrengifo commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/SampleCardsFDSView.java:
```diff
@@ -25,7 +25,7 @@
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.CUSTOMIZED_FDS_SAMPLE,
 	service = FDSView.class
 )
 public class SampleCardsFDSView extends BaseCardsFDSView {
```
Does this class name should reference the customized sample, like in `CustomizedSampleTableFDSView`?

markocikos replied:
Yeah, that will make it more consistent. I'll also change the constants to match.

beltranrengifo commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/SampleListFDSView.java:
```diff
@@ -25,7 +25,7 @@
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.CUSTOMIZED_FDS_SAMPLE,
 	service = FDSView.class
 )
 public class SampleListFDSView extends BaseListFDSView {
```
Does this class name should reference the customized sample, like in CustomizedSampleTableFDSView?

beltranrengifo commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/SampleTimelineFDSView.java:
```diff
@@ -25,7 +25,7 @@
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.CUSTOMIZED_FDS_SAMPLE,
 	service = FDSView.class
 )
 public class SampleTimelineFDSView extends BaseTimelineFDSView {
```
Does this class name should reference the customized sample, like in CustomizedSampleTableFDSView?

beltranrengifo reviewed (commented):
Hi @markocikos ! 
Thank you for the progress you made here 🙌 
I've left you some minor comments inline

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/MinimumSampleTableFDSView.java:
```diff
@@ -0,0 +1,64 @@
+	property = "frontend.data.set.name=" + FDSSampleFDSNames.MINIMUM_FDS_SAMPLE,
+	service = FDSView.class
+)
+public class MinimumSampleTableFDSView extends BaseTableFDSView {
```
There is currently no way to disable this, we are [unconditionally adding](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/TableHead.js#L78-L84) the `FieldsSelectorDropdown` component. Potentially changing this is a product call, see https://liferay.slack.com/archives/C02CH9Q6Q21/p1650366641759489

markocikos commented:
Force-pushed! Changes:
- Renamed model files to match FDS ID. More specifically, if a file contains `property = "frontend.data.set.name=*`, it has a prefix with that name.
- Fixed tabs inclusion logic, so that only one FDS loads with tab.

markocikos commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/2179/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/2179/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f9d0952034647b50038707cecf6c90cd0403f2e1">f9d0952034647b50038707cecf6c90cd0403f2e1</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/markocikos/liferay-portal/tree/LPS-151456">LPS-151456</a><br/>Branch GIT ID: <a href="https://github.com/markocikos/liferay-portal/commit/96158bd390ae6f11a0e96786368a1c85513c8732">96158bd390ae6f11a0e96786368a1c85513c8732</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/2179/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/2179/jenkins-report.html">here</a>.</h5></details></html>

markocikos commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/2478/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2478/jenkins-report.html">here</a>.</b></p>

beltranrengifo reviewed (approved):
LGTM!

markocikos commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

The pull request will automatically be forwarded to the user `brianchandotcom` if the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
`ci:test:relevant`
`ci:test:sf`

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-14.liferay.com/job/forward-pullrequest/331//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to https://github.com/brianchandotcom/liferay-portal/pull/116403
[Console](https://test-1-14.liferay.com/job/forward-pullrequest/331//consoleFull)
